nodeJS/express_and_mongoose/express_102_crud_and_mvc.md

```diff
@@ -19,58 +19,58 @@ This section contains a general overview of topics that you will learn in this l
 
 CRUD is a concept that comes up a lot in web development, and it's the type of thing that might show up in interview questions so it's worth taking a little time to make sure you understand what it refers to.
 
-CRUD stands for: <span id="crud">_Create, Read, Update_ and _Delete_</span>. These are the four basic functions that you will be building into your database driven apps. If you are designing a CRUD interface that means that users can expect to be able to do these 4 things to items in the database (providing they have the appropriate permissions of course).
+CRUD stands for: <span id="crud">*Create, Read, Update* and *Delete*</span>. These are the four basic functions that you will be building into your database driven apps. If you are designing a CRUD interface that means that users can expect to be able to do these 4 things to items in the database (providing they have the appropriate permissions of course).
```

Comment:
```suggestion
CRUD stands for: *Create, Read, Update* and *Delete*. These are the four basic functions that you will be building into your database driven apps. If you are designing a CRUD interface that means that users can expect to be able to do these 4 things to items in the database (providing they have the appropriate permissions of course).
```
Not something you've made but I just noticed that we have a redundant span here, as the `### CRUD` heading itself directly above already generates an `h3#crud`, so the span isn't actually needed for scroll either (since it's not like it's a long way away from the heading). Mind removing it while we're at it?

Everything else looks :man_cook: :pinched_fingers:

Reply:
Done!